cmd/frontend/graphqlbackend/site_update_check.go

```diff
@@ -8,19 +8,19 @@ import (
 )
 
 func (r *siteResolver) UpdateCheck(ctx context.Context) (*updateCheckResolver, error) {
-	// 🚨 SECURITY: Only site admins can check for updates.
-	if err := backend.CheckCurrentUserIsSiteAdmin(ctx); err != nil {
-		return nil, err
-	}
+	// 🚨 SECURITY: Only site admins can check for updates but users may see notifications
 	return &updateCheckResolver{
-		last:    updatecheck.Last(),
-		pending: updatecheck.IsPending(),
+		last:        updatecheck.Last(),
+		pending:     updatecheck.IsPending(),
+		IsSiteAdmin: backend.CheckCurrentUserIsSiteAdmin(ctx) == nil,
```

Comment:
```suggestion
		IsSiteAdmin: backend.CheckCurrentUserIsSiteAdmin(ctx) == backend.ErrMustBeSiteAdmin,
```

However, I don't think we want unauthenticated requests to this resolver to work. Can you verify this?

Reply:
Good catch, yes I want this to only be called by an authenticated users